Common/Model/Subscriptions.cs

```diff
@@ -81,6 +81,23 @@ public class Subscription : SubscriptionWithLease {
         }
     }
 
+    public class SubscriptionWithHubURL : Subscription

```

Comment:
So the reason I created it was because just using Subscription the HubURL property would not be sent to the client because of the JsonIgnore attribute. If I remove that attribute then the HubURL is part of Subscription for all the other interactions (subscription requests and verifications). I agree it isn't elegant or an ideal solution which is why I opened up issue #53 which aims at figuring out a better way to model the Subscription object for all cases so that we don't need to create a new class just because we do/don't want to serialize a particular property.